packages/client/src/objectSet/ObjectSetListenerWebsocket.test.ts

```diff
@@ -126,8 +135,15 @@ describe("ObjectSetListenerWebsocket", async () => {
         objectSetExpiryMs: OBJECT_SET_EXPIRY_MS,
       });
 
+      listenerPromise = new Promise((resolve) => {
+        listenerPromiseResolveFn = resolve;
+      });
+
       listener = {
-        onChange: vi.fn(),
+        onChange: vi.fn((o) => {
+          listenerPromiseResolveFn();
```

Comment:
I need to do this to make sure the test doesn't end before we receive the update